pgoapi/pgoapi.py

```diff
@@ -308,6 +358,19 @@ def spin_all_forts_visible(self):
     def return_to_start(self):
         self.set_position(*self._origPosF)
 
+    def walk_to_fort(self, fort_data, directly=False):
+        fort = fort_data[0]
+        self.log.info("Walking to fort at  http://maps.google.com/maps?q=%s,%s", fort['latitude'],
+                        fort['longitude'])
+        if directly is True:
```

Comment:
I was trying to maintain the signature of `walk_to` by not introducing `directly` kwarg on it but it would be backwards compatible to add it so I guess I can just do that